device/bgp/bgp.yaml

```diff
@@ -82,3 +82,15 @@ components:
           type: array
           items:
             $ref: './bgpv6routerange.yaml#/components/schemas/Bgp.V6RouteRange'
+        v4_srte_policies:
+          description: >-
+            Segment Routing (SR) Policy for IPv4 Address family indicator (AFI).
```

Comment:
Make the first alphabet capital for “family” and “indicator”. It should be “IPv4 Address Family Indicator (AFI)”.

Reply:
Done